pkg/analysis/nomaps/testdata/src/a/a.go

```diff
@@ -0,0 +1,22 @@
+package a
+
+type DefinedType map[string]Component
+
+type TypeAlias = map[string]Component
```

Comment:
Make sure to also check pointers. Take a look at the max length tests, there's lots of examples in there for how we combine pointers and aliases and pointer aliases please